static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -356,3 +358,18 @@ export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
   }
   return false;
 }
+/**
+ * Converts the disease parent data from a flat array to a tree for tree visualization
+ * @param data the disease ontology data pertaining to the disease of interest
+ * @returns an array consisting of disease name and a sub-array of its children
```

Comment:
we're not returning an array, just returning the topmost parent I believe

Reply:
Yes, by array I meant topmost parent and its children but you're right, my wording was a bit unclear. Fixed it!